bindings/rust/s2n-tls-sys/README.md

```diff
@@ -1,3 +1,43 @@
 This crates provides low level rust bindings for [s2n-tls](https://github.com/aws/s2n-tls) which are autogenerated with [bindgen](https://github.com/rust-lang/rust-bindgen)
 
-This crate is not intended for direct consumption by end consumers. Interested developers should instead look at the [s2n-tls](https://crates.io/crates/s2n-tls) or [s2n-tls-tokio](https://crates.io/crates/s2n-tls-tokio) crates. These  provide higher-level, more ergonomic bindings than the `s2n-tls-sys` crate.
\ No newline at end of file
+This crate is not intended for direct consumption by end consumers. Interested developers should instead look at the [s2n-tls](https://crates.io/crates/s2n-tls) or [s2n-tls-tokio](https://crates.io/crates/s2n-tls-tokio) crates. These  provide higher-level, more ergonomic bindings than the `s2n-tls-sys` crate.
+
+# Bring Your Own libs2n
+
+The `s2n-tls-sys` crate contains the raw C code of `s2n-tls`. By default, it follows this build process:
+
+1. Uses the system C compiler to build `libs2n.a`
+2. Links the built `libs2n.a` to the Rust bindings
+3. Links against `aws-lc` through the `aws-lc-rs` crate
```

Comment:
Nit: I'd move this above the `bring your own libs2n` heading, because it it generally useful information.